### PR TITLE
Add support for HasRemoteParent, Status and MessageEvents to OC proto translator

### PR DESCRIPTION
**Pull request by subnova-etsy**
Adds support for setting:
* HasRemoteParent field from the SameProcessAsParentSpan OC field (inverted)
* Status from the Status.Code OC field
* MessageEvents from OC Annotations

Adds a test case for the translator.

### CODE REVIEW

File: honeycomb/translator.go [hunk 1]
```diff
@@ -143,11 +160,12 @@ func OCProtoSpanToOTelSpanData(span *tracepb.Span) (*trace.SpanData, error) {
 	copy(spanData.ParentSpanID[:], span.GetParentSpanId()[:])
 	spanData.Name = getSpanName(span)
 	spanData.SpanKind = oTelSpanKind(span.GetKind())
-	spanData.ChildSpanCount = int(span.GetChildSpanCount().GetValue())
```
Comment (paulosman) [inline honeycomb/translator.go]: Good catch, thanks :)

**Comment by lizthegrey** [comment]
Oh, crud, if I'd seen this was pending I would have held v0.3.0 for this. Want to roll v0.3.0.1 for this, @paulosman?

**Comment by paulosman** [comment]
Yeah, sounds good. I'll cut it later today.

**Comment by Aneurysm9** [comment]
`v0.3.1`, please. Go won't pick up modules with tags that don't match the SemVer spec.  See https://github.com/open-telemetry/opentelemetry-go/blob/master/tag.sh#L45 for a regexp that can validate tag names.

**Pull request merged**